### ISSUE TITLE
Can only export once per session, other exports do nothing

### ISSUE THREAD

**Issue by mriva** [dbcli/mycli#898]
If I use `\o outputfile.csv` then run a query, I get the output in the file only the first time I try it. If I run the commands other times they do nothing until I quit and restart mycli

mycli version: 1.20.1

**Comment by meeuw** [comment]
Hi @mirva! This is expected behavior, if you look at:
```
\?
...
| \once       | \o [-o] filename           | Append next result to an output file (overwrite using -o). |
```

You have to repeat `\o` for every query you want to log. I found this useful for exporting the output from queries to multiple files.

If you want everything logged you should use `tee`.

**Comment by mriva** [comment]
I know and I do that, but it doesn't work after the first time.

**Comment by meeuw** [comment]
Hm that's weird. You aren't wrinting to the same file without using the `-o` overwrite option right?

I've tried the following and that seems to work:
```
mysql root@localhost:(none)> \o hier                                                                                                                                                  
Time: 0.000s
mysql root@localhost:(none)> select user from mysql.user                                                                                                                              
+---------------+
| user          |
+---------------+
| root          |
| mysql.session |
| mysql.sys     |
| root          |
+---------------+
4 rows in set
Time: 0.009s
mysql root@localhost:(none)> \o daar                                                                                                                                                  
Time: 0.000s
mysql root@localhost:(none)> select user from mysql.user                                                                                                                              
```

**Comment by mriva** [comment]
No I'm writing to different files, here's what happens:

```
mysql root@127.0.0.1:(none)> \o ~/tmp/output/one                                                                                                                                  
Time: 0.000s
mysql root@127.0.0.1:(none)> SHOW DATABASES;                                                                                                                                      
+--------------------+
| Database           |
+--------------------+
| information_schema |
| mysql              |
| performance_schema |
...
```

``` ~/tmp/output
$ ll
total 4,0K
-rw-rw-r-- 1 teoss teoss 299 set 18 16:50 one
```

```mysql root@127.0.0.1:(none)> \o ~/tmp/output/two                                                                                                                                  
Time: 0.000s
mysql root@127.0.0.1:(none)> SHOW DATABASES;                                                                                                                                      
+--------------------+
| Database           |
+--------------------+
| information_schema |
| mysql              |
| performance_schema |
...
```

``` ~/tmp/output
$ ll
total 4,0K
-rw-rw-r-- 1 teoss teoss 299 set 18 16:50 one
```

**Comment by meeuw** [comment]
I've tried to use `1.20.1` but it fails on pymysql (I guess the dependencies are wrong).

Can you update to the latest version of mycli?

**Comment by bamanzi** [comment]
I'm suffering frfom the same problem here, also with `1.20.1`.

Sometimes no new file generated, sometimes all I got is a single line file (filled with `+---------------+--------` but nothing else).

Problem insisted even I restarted `mycli` (i.e. no correct file generated).

Because we're using python-2.7 on production environment, we can't upgrade it to the latest version. But I'll try checking this with more cases.

**Comment by pasenor** [comment]
It was fixed in 1.21.0 https://github.com/dbcli/mycli/issues/798
Unfortunately no backport for python 2.7

**Comment by bamanzi** [comment]
I'm suffering frfom the same problem here, also with `1.20.1`.

Sometimes no new file generated, sometimes all I got is a single line file (filled with `+---------------+--------` but nothing else).

Problem insisted even I restarted `mycli` (i.e. no correct file generated).

Because we're using python-2.7 on production environment, we can't upgrade it to the latest version. But I'll try checking this with more cases.
